contracts/libraries/Welford.sol

```diff
@@ -37,11 +37,10 @@ library Welford {
         int256 _m2 = int256(curM2).add(delta.mul(delta2));
 
         require(_count > 0, "count<=0");
-        require(_mean >= 0, "mean<0");
         require(_m2 >= 0, "m2<0");
```

Comment:
Not possible for m2 < 0 because we multiply either '+ * +` or '- * -', which results in a positive number